File: 10 - Python/02 - Estrutura Condicional/Exercício 05.py
#Exercício - Troco verificado

preco = float(input("Preço unitário do produto: R$ "))
quantidade = int(input("Quantidade comprada: "))
dinheiro = float(input("Dinheiro recebido: "))

troco = dinheiro - (preco * quantidade)

if troco >= 0:
    print("Troco = R$", troco)
else:
    print(f"Dinheiro insuficiente. Faltam R$ {abs(troco)}")

File: 07 - C++/Exercícios/02 - Estrutura Condicional/Exercício 05.cpp
//Exercício - Troco Verificado

#include <iostream>
#include <iomanip>
#include <cmath>

using namespace std;

int main() {

    int quantidade;
    double preco, dinheiro, troco;

    cout << "Preco unitario do produto: ";
    cin >> preco;

    cout << "Quantidade comprada: ";
    cin >> quantidade;

    cout << "Dinheiro recebido: ";
    cin >> dinheiro;

    troco = dinheiro - (preco * quantidade);

    cout << fixed << setprecision(2);

    if (troco >= 0){
        cout << "Troco = R$" << troco << endl;
    }
    else{
        troco = abs(troco);
        cout << "Dinheiro insuficiente. Faltam R$" << troco << endl;
    } 

    return 0;

}